content/mactynow/drone-s3cache/index.md

```diff
@@ -0,0 +1,110 @@
+---
+date: 2017-06-29T00:00:00+00:00
+title: Drone-s3cache
+author: mactynow
+tags: [ s3, cache ]
+repo: mactynow/drone-s3cache
+logo: amazon_s3.svg
+image: quay.io/honestbee/drone-kubernetes
+---
+
+{{% alert error %}}
```

Comment:
Can you change this to warning?

Reply:
IMHO this alert should be dropped at all :D